doc/Overview/Localization/LocalizationOverview.md

```diff
@@ -1,46 +1,70 @@
 
-An implementation of `IStringLocalizer` (`ResourceLoaderStringLocalizer`) is registered as service.
+## Set up localization
 
 ```csharp
-private IHost Host { get; }
-
-public App()
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLocalization()
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => 
+        {
+            host.UseLocalization()
+        });
+...
+```
+
+An implementation of `IStringLocalizer` (`ResourceLoaderStringLocalizer`) will be registered as a service. This service offers a consistent way to resolve localized strings. Behind the scenes, it will automatically use `ResourceManager` on Windows and `ResourceLoader` on other platforms.
+
+### Adding language specific resources
+
+The `ResourceLoaderStringLocalizer` will look for `.resw` files in folders corresponding to the well-known language tag (eg en-US). For example, if the current culture is `en-US`, the `ResourceLoaderStringLocalizer` will look for `.resw` files in the `en-US` folder. If the current culture is `fr-FR`, the `ResourceLoaderStringLocalizer` will look for `.resw` files in the `fr-FR` folder.
+
+To add a new resource file, right click on the project and select **Add > New Item...**. Select **Resource File (.resw)** and name it according to the language tag (eg `en-US.resw`). Resource files have a key-value pair structure. The key is used to identify the resource and the value is the localized text. The key contains a name which corresponds to the `x:Uid` property of the element in XAML. The value contains the localized text.
+
+For example, if the `x:Uid` property of a `TextBlock` is `MyTextBlock`, the key in the resource file should be `MyTextBlock.Text`. The value of the key is the localized text.
+
+### Resolving localized strings
+
+Once locale specific resources are included, the localization feature can be used to resolve those localized texts.
+
+In XAML, assigning the `x:Uid` property a value looks like this:
+
+```xml
+<TextBlock x:Uid="MyTextBlock" />
 ```
 
-We use `IStringLocalizer` to resolve those localized texts
+However, the `x:Uid` property is not required to resolve localized strings. The `IStringLocalizer` service can be resolved from the service provider.
 
 ```csharp
 var stringLocalizer = serviceProvider.GetService<IStringLocalizer>();
+```
+
+Localized strings can be resolved using the indexer on the IStringLocalizer as a dictionary. This indexer takes a key and returns the localized string.
 
-// Using IStringLocalizer as a dictionary
+```csharp
 string myString = stringLocalizer["MyKey"];
+```
+
+LocalizedString objects can also be resolved from the IStringLocalizer. These objects contain the localized string and a boolean indicating whether the resource was found.
```

Comment:
Why is this useful? Please elaborate on why it makes sense to use this, rather than simply casting to string